common/templates/errorDialog.controller.js

```diff
@@ -0,0 +1,15 @@
+(function() {
+    'use strict';
+
+    angular.module('chaise.errors')
+
+    .controller('ErrorDialogController', ['$uibModalInstance', 'params', function ConfirmDeleteController($uibModalInstance, params) {
```

Comment:
For consistency, it might be better to change `ConfirmDeleteController` to `ErrorDialogController`.